assets/js/map.js

```diff
@@ -122,19 +125,20 @@ const MapBase = {
       })
     );
 
-    L.control.zoom({
-      position: 'bottomright',
+    new L.Control.ZoomEx({
+      position: "bottomright",
+      className: "leaflet-zoomex-rightbottom",
```

Comment:
This zoom slider should be hidden in preview mode.
To enable preview mode use "?q=trains" query string.

Also a little suggestion. Can we make it a bit darker to match rest of the UI? It stands out right now. If you make it darker, maybe you will need to decrease the opacity too a little bit. Up to you

Reply:
Those are default styles and yeah, I also feel it doesn't quite match the overall tone of map :/ You take over, then..